graylog2-web-interface/src/components/indices/rotation/TimeBasedRotationStrategyConfiguration.jsx

```diff
@@ -136,14 +136,14 @@ class TimeBasedRotationStrategyConfiguration extends React.Component {
         <Input id="rotate-empty-index-sets"
                labelClassName="col-sm-3"
                wrapperClassName="col-sm-9"
-               label="Empty index sets">
+               label="Empty index set">
           <Input id="rotate-empty-index-sets-checkbox"
                  type="checkbox"
-                 ref={(rotateEmptyIndexSetsRef) => { this.inputs.rotate_empty_index_sets = rotateEmptyIndexSetsRef; }}
-                 label="Rotate empty index sets"
-                 onChange={this._onRotateEmptyIndexSetsUpdate('rotate_empty_index_sets')}
-                 checked={rotateEmptyIndexSets}
-                 help="Apply the rotation strategy on empty index sets." />
+                 ref={(rotateEmptyIndexSetRef) => { this.inputs.rotate_empty_index_set = rotateEmptyIndexSetRef; }}
+                 label="Rotate empty index set"
+                 onChange={this._onRotateEmptyIndexSetUpdate('rotate_empty_index_set')}
+                 checked={rotateEmptyIndexSet}
+                 help="Apply the rotation strategy even when all indexes are empty (not recommended)." />
```

Comment:
```suggestion
                 help="Apply the rotation strategy even when the index set is empty (not recommended)." />
```

Sorry for this new suggestion again, but after reading my previous one again, I think this one fits much better.